### PR TITLE
Dirkhh for bstoeger raw

### PR DESCRIPTION
**Pull request by dirkhh**
### Describe the pull request:

- [x] Code cleanup
- [x] Build system change

### Pull request long description:

- small `cmake` change to allow output of `pkg_config_library()` macro to be shown in CICD build output
- small fix to `get-version.sh` so it doesn't clutter the `cmake` output with strange errors about invalid commit names
- add libraw to various CICD builds (the macOS one is tested, the others I need to double check - hence this PR)

### Mentions:

@bstoeger @mikeller

### CODE REVIEW

**Review by bstoeger — APPROVED** [review]
Very nice, thanks!
(Not that I understand any of that, but it looks reasonable.)

**Comment by dirkhh** [comment]
> Very nice, thanks! (Not that I understand any of that, but it looks reasonable.)

now you know how I feel with many of the C++ PRs 🤣
This is a PR against your branch as that's where it makes sense - so I'll leave it to you if / how / when you want to merge this.

**Comment by dirkhh** [comment]
oh - the logic of when the builds run surprised me here. I had hoped that this PR would still create test builds so we can see if at least some of the changes actually work (the one for Copr won't be triggered until this gets merged, sadly)

File: .github/workflows/mac.yml
Comment (mikeller) [inline .github/workflows/mac.yml:27]: Does this actually result in the entire libraw library being included in the .dmg that is built, or does this just add the library stubs so that Subsurface is built against libraw?
i.e. what happens when a version of Subsurface built in this way is run on a Mac that does not have the homebrew `libraw` installed when the user tries to add a RAW image?
The same applies for other distribution channels like COPR as well, where I think we need to declare a dependency on libraw in the manifest.

Comment (dirkhh) [reply .github/workflows/mac.yml:27]: macOS binaries are very much like AppImages or Snaps. They contain all of the libraries that they depend on (with the exception of what is considered 'core system libraries'). So this treats `libraw` just like we treat `libxslt` or `libjpg` or `libpng`. They all get bundled with the `Subsurface.app`

Comment (mikeller) [reply .github/workflows/mac.yml:27]: TIL

**Pull request merged**
